### PR TITLE
search bar task

### PR DESCRIPTION
**Pull request by krolich500grn**
- [DEMO LINK](https://krolich500grn.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://krolich500grn.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+.input:hover {
+  background-color: #fff;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  font-weight: bold;
+  background: url(./images/Search.svg) no-repeat 25px 24px / 19px 19px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%) ;
+  border-radius: 3px;
```
Comment (solaryasha) [inline src/style.css:51]: I guess, you could avoid this change of 1px

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  border-color: #e1e7ed;
+}
+
+.input--small:focus {
+  background: url(./images/Search.svg) no-repeat 12px 14px / 11px 11px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%) ;
+}
+
+[data-qa="big"] {
+  margin: 20px 0;
+}
```
Comment (solaryasha) [inline src/style.css:61]: Why do you use it ? :)

Comment (solaryasha) [reply src/style.css:61]: You class selectors for styling

**Review by solaryasha — APPROVED** [review]
Good job !

File: src/style.css (hunk 2 above)
Comment (krolich500grn) [reply src/style.css:61]: okay, i'll use it next time :)

**Pull request closed without merging**